server/db/seed/Gig.js

```diff
@@ -0,0 +1,92 @@
+// title: DataTypes.STRING,
+// type: DataTypes.ENUM('wedding', 'birthday', 'business'),
+// date: DataTypes.DATE,
+// start_time: DataTypes.TIME,
+// end_time: DataTypes.TIME,
+// complexity: DataTypes.INTEGER,
+// health: DataTypes.INTEGER
+
+function randomItemFromArray(array){
+  return array[Math.floor(Math.random()*array.length)];
+}
+
+var moment = require('moment');
+
+var titles = [
+  'George Washington Event',
+  'John Adams Event',
+  'Thomas Jefferson Event',
+  'James Madison Event',
+  'James Monroe Event',
+  'John Quincy Event',
+  'Andrew Jackson Event',
+  'Martin Van Event',
+  'William Henry Event',
+  'John Tyler Event',
+  'James Knox Event',
+  'Zachary Taylor Event',
+  'Millard Fillmore Event',
+  'Franklin Pierce Event',
+  'James Buchanan Event',
+  'Abraham Lincoln Event',
+  'Andrew Johnson Event',
+  'Ulysses Simpson Event',
+  'Rutherford Birchard Event',
+  'James Abram Event',
+  'Chester Alan Event',
+  'Grover Cleveland Event',
+  'Benjamin Harrison Event',
+  'Grover Cleveland Event',
+  'William McKinley Event',
+  'William Howard Event',
+  'Woodrow Wilson Event',
+  'Warren Gamaliel Event',
+  'Calvin Coolidge Event',
+];
+
+var types = [
+  'Wedding',
+  'Birthday',
+  'Batmitzvah',
+  'Party',
+  'Graduation',
+  'Launch party',
+  'Bachelors party',
+  'Bachelorette party'
+];
+
+var today = moment();
+
+var dates = [
+  // today.subtract(1, 'days').toISOString(),
```

Comment:
realized i was adding more data then necessary, just wanted to keep it simple for now